helm/blueapi/templates/statefulset.yaml

```diff
@@ -69,39 +56,83 @@ spec:
             mountPath: "/config"
             readOnly: true
           - name: scratch-host
-            mountPath: {{ .Values.initContainer.scratch.root }}
+            mountPath: {{ .Values.initContainer.config.scratch.root | required "initContainer enabled but config does not contain scratch.root" }}
             mountPropagation: HostToContainer
           - name: venv
             mountPath: /artefacts
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}
+          {{- with .Values.securityContext }}
           securityContext:
-            {{- toYaml .Values.securityContext | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: {{ ((.Values.worker).api).port | default 8000 }}
+              protocol: TCP
+          {{- with .Values.livenessProbe }}
+          livenessProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.readinessProbe }}
+          readinessProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.resources }}
```

Comment:
Should: I think these were removed because they caused problems, are you trying to fix with #884?

Reply:
Yes, that's the purpose of 884.

Reply:
#884 has now gone in, so the probes now get `/healthz` which returns a 200 code